linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+#script pseudocode
+#hint: `systemctl status docker || ...`
+#if docker_deamon is not running
+#	start docker_deamon
```

Comment:
* Remove unused comments
* Please review shell coding style https://www.notion.so/jarviscanada/Shell-Style-Guide-764a3ac0c9044fdb844b55420d1c7594

Reply:
remove all hints and leave necessary comments only